kong/runloop/handler.lua

```diff
@@ -46,6 +47,7 @@ local get_phase         = ngx.get_phase
 local subsystem         = ngx.config.subsystem
 local clear_header      = ngx.req.clear_header
 local http_version      = ngx.req.http_version
+local resp_get_headers  = ngx.resp and ngx.resp.get_headers
```

Comment:
seems we dont need to check ngx.resp, because I found you directly use resp_get_headers in the following code, which means you assure that resp_get_headers is always non-nil

Reply:
@chobits if we do not check `ngx.resp`, Kong refuses to start.

At the startup phase, `ngx.resp` is `nil`.

Reply:
bin/kong need to check it.